code/player/TTTPlayer.Confirmation.cs

```diff
@@ -11,6 +11,8 @@ partial class TTTPlayer
 
         public bool IsConfirmed = false;
 
+        public bool IsMissingInAction = false;
```

Comment:
Should we just have an enum for the state of the player? instead of having separate bools for each?

i.e
```csharp
enum PlayerStatus {
	Confirmed
	MIA
	Alive
	Dead
}
```

Reply:
Sadly, that doesn't work as a dead player can be confirmed. And an alive player can be alive and MIA.

Reply:
How many possible combinations are there? There is the possibility of either...

1. Just having all the possible combinations in the enums
2. Use this https://stackoverflow.com/questions/19582477/combining-enum-values-with-bit-flags

Reply:
Hmmm..  maybe for another PR. It's not as simple keeping track of them properly as Alive and Dead arise from the LifeState which is handled automatically and is synced to all clients as well.

Reply:
kk throw up a TODO / task and I'll check it out.